### PR TITLE
Examples changes for dynamic display width and height

### PR DESCRIPTION
**Pull request by jposada202020**
As stated in the title. In addition, examples addition to the API readthedocs

### CODE REVIEW

**Comment by kmatch98** [comment]
Makes sense to me.

**Comment by jposada202020** [comment]
@kmatch98 Thanks for reviewing Kmtach :)

File: examples/display_button_customfont.py [hunk 1]
```diff
@@ -11,6 +11,8 @@
 import adafruit_touchscreen
 from adafruit_button import Button
 
+display = board.DISPLAY
```
Comment (FoamyGuy) [inline examples/display_button_customfont.py:17]: Can you include this comment about the display setup here please
```
# use built in display (PyPortal, PyGamer, PyBadge, CLUE, etc.)
# see guide for setting up external displays (TFT / OLED breakouts, RGB matrices, etc.)
# https://learn.adafruit.com/circuitpython-display-support-using-displayio/display-and-display-bus
display = board.DISPLAY
```
This helps folks discover the relevant guide page for setting up external displays if that is what they are using.
```

**Review by FoamyGuy — COMMENTED** [review]
Thanks for these improvements @jposada202020.

I have one small suggestion, but this looks good to me beyond that.

File: examples/display_button_customfont.py (hunk 1 above)
Comment (jposada202020) [reply examples/display_button_customfont.py:17]: @FoamyGuy Thanks, I added the note in the examples, and add the MagTag to the list also

Comment (jposada202020) [reply examples/display_button_customfont.py:17]: https://github.com/adafruit/Adafruit_CircuitPython_Display_Button/pull/30/commits/dd4fea43a45dbf355bf12b84d2ff35715d9cfe0d

**Review by FoamyGuy — APPROVED** [review]
Looks good to me. Tested successfully on: 
```
Adafruit CircuitPython 6.2.0-rc.0 on 2021-04-01; Adafruit PyPortal with samd51j20
```

**Pull request merged**